test/controllers/nonsipp/landorpropertydisposal/LandPropertyDisposalCYAControllerSpec.scala

```diff
@@ -123,4 +143,58 @@ class LandPropertyDisposalCYAControllerSpec extends ControllerBaseSpec {
       )
     }
   }
+
+  "LandPropertyDisposalCYAController in view only mode" - {
+
+    val currentUserAnswers = userAnswers
+      .unsafeSet(FbVersionPage(srn), "002")
+
+    val previousUserAnswers = currentUserAnswers
+      .unsafeSet(FbVersionPage(srn), "001")
+
+    act.like(
```

Comment:
It might be worthwhile to have a `.withName` for this, but it's very minor, so please consider adding it only if there are more changes needed based on QA.